signac_dashboard/templates/login.html

```diff
@@ -0,0 +1,25 @@
+{% extends "layout.html" %}
+
+{% block title %}{% endblock %}
+{% block subtitle %}{% endblock %}
+{% block view_options %}{% endblock %}
+{% block module_selector %}{% endblock %}
+{% block searchbar %}{% endblock %}
+
+{% block panels %}
+
+<div class="container column is-8">
+  <form method="POST" action="/login" class="box">
+    <h4 class="title">Logged out</h4>
+    <input class="block input is-medium" type="password" id="token" name="token" placeholder="Token">
+    <button type="submit" class="block button is-primary">Submit</button>
+    <p>
+      Token authentication is enabled. Copy the token from the command line where dashboard was run.
```

Comment:
Avoid using the word "dashboard" to mean "signac-dashboard" the application.
```suggestion
      Token authentication is enabled. Copy the token from the command line where signac-dashboard was run.
```

Reply:
Good point